### PR TITLE
docs: Provide consistent docs for trigger authentication

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Provide consistent docs for trigger authentication across all scalers.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

**Comment by tomkerkhove** [comment]
@JorTurFer Would you mind taking a look as well?

File: content/docs/2.0/scalers/redis-streams.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.0/scalers/redis-streams.md:73]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.1/scalers/redis-cluster-streams.md [hunk 2]
```diff
@@ -66,6 +66,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.1/scalers/redis-cluster-streams.md:71]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.1/scalers/redis-streams.md [hunk 3]
```diff
@@ -68,6 +68,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.1/scalers/redis-streams.md:73]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.2/scalers/redis-cluster-streams.md [hunk 4]
```diff
@@ -66,6 +66,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.2/scalers/redis-cluster-streams.md:71]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.2/scalers/redis-streams.md [hunk 5]
```diff
@@ -68,6 +68,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.2/scalers/redis-streams.md:73]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.3/scalers/redis-cluster-streams.md [hunk 6]
```diff
@@ -66,6 +66,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.3/scalers/redis-cluster-streams.md:71]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.3/scalers/redis-streams.md [hunk 7]
```diff
@@ -68,6 +68,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.3/scalers/redis-streams.md:73]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.4/scalers/redis-cluster-streams.md [hunk 8]
```diff
@@ -66,6 +66,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.4/scalers/redis-cluster-streams.md:71]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.4/scalers/redis-streams.md [hunk 9]
```diff
@@ -68,6 +68,10 @@ Some parameters could be provided using environmental variables, instead of sett
 
 The scaler supports two modes of authentication:
 
+**Password Authentication:**
+
+- `password` - Redis password to authenticate with
```
Comment (JorTurFer) [inline content/docs/2.4/scalers/redis-streams.md:73]:
```suggestion
- `password` - Redis password to authenticate with.
```

File: content/docs/2.4/scalers/solace-pub-sub.md [hunk 10]
```diff
@@ -37,18 +39,22 @@ triggers:
...
 ### Authentication Parameters
+
 You can use TriggerAuthentication CRD to configure the username and password to connect to the management endpoint.
 
 **Username and Password based authentication:**
-- `username` - Required. The username to use to connect to the Solace PubSub+ Event Broker's SEMP endpoint.
-- `password` - Required. The password to use to connect to the Solace PubSub+ Event Broker's SEMP endpoint.
+
+- `username` The username to use to connect to the Solace PubSub+ Event Broker's SEMP endpoint.
+- `password` The password to use to connect to the Solace PubSub+ Event Broker's SEMP endpoint.
```
Comment (JorTurFer) [inline content/docs/2.4/scalers/solace-pub-sub.md:54]:
```suggestion
- `username` - The username to use to connect to the Solace PubSub+ Event Broker's SEMP endpoint.
- `password` - The password to use to connect to the Solace PubSub+ Event Broker's SEMP endpoint.
```

**Review by JorTurFer — CHANGES_REQUESTED** [review]
Nice job! only small nits

**Review by JorTurFer — APPROVED** [review]
LGTM!

**Pull request merged**